sql_upgrade.php

```diff
@@ -86,14 +97,19 @@
   require("acl_upgrade.php");
```

Comment:
Looked at this script, and this is the only script that should modify the acl version(and that is all it modifies). It's always run in context of OpenEMR(either is run here or is run as a link off of admin.php), so should be able to use the standard entity.